spec/support/database.rb

```diff
@@ -27,8 +27,13 @@ def setup
 
   def migrate
     ActiveRecord::Migration.verbose = false
-    ActiveRecord::Base.configurations = YAML.safe_load(ERB.new(IO.read("#{dirname}/database.yml")).result)
-    ActiveRecord::Base.establish_connection ActiveRecord::Base.configurations[adapter]
+    databaseyml = YAML.safe_load(ERB.new(IO.read("#{dirname}/database.yml")).result)
+    if ActiveRecord::VERSION::STRING >= "6.1"
```

Comment:
Does it make sense to flip this conditional or flip the others?  I think most of the others do `if < 6.1`.  I'd be fine with having this format above and change the others so it's obvious that 6.1+ is the expected outcome going forward and have the `else` handle the less expected cases.